conformance/system_macros/annotate.ion

```diff
@@ -0,0 +1,9 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Test Cases:
+// annotate can be invoked by name, address, qualified address, qualified name, binary address, and binary system e-expression op code.
+// annotate adds annotations to a value
+// if the value has existing annotations, they are preserved and annotate prepends annotations to the existing annotations of the value
+// the first argument must be zero or more non-null text values ($0 is allowed)
+// the second argument must be a single expression
```

Comment:
```suggestion
// the second argument must be a singleton expression
```
or
```suggestion
// the second argument must expand to a single expression
```
?

Reply:
Right, but if you say "it must be a single expression" that doesn't preclude something like `(:values 1 2 3)`--one expression that expands to multiple values.